src/admin/participants/routes.js

```diff
@@ -7,7 +7,33 @@ const RouteConfig = require('../route-config')
 
 const tags = ['api', 'participants']
 const nameValidator = Joi.string().alphanum().min(3).max(30).required().description('Name of the participant')
-const passwordValidator = Joi.string().regex(/^[a-zA-Z0-9]{3,30}$/).required().description('Password for the participant')
+// const passwordValidator = Joi.string().regex(/^[a-zA-Z0-9]{3,30}$/).required().description('Password for the participant')
+const currencyValidator = Joi.string().allow([
```

Comment:
Shouldn't this be loaded from the DB?

I think we need to add a story for this. Although perhaps its not necessary once we move to the API Design first approach, and use a RAML/Swagger to generate the interfaces classes and validations for us.

Note: The approach can be discussed/determined as part of that. If its the later approach then the story can be closed with the decision documented. I have created the following story for this purpose: https://github.com/mojaloop/project/issues/283  <-- FYI: @elnyry @ksatya77 @NicoDuvenage @wdcmb

Reply:
no change made here. awaiting currency validation story